pkg/operator/ceph/cluster/osd/osd.go

```diff
@@ -314,6 +319,40 @@ func (c *Cluster) Start() error {
 	return nil
 }
 
+func (c *Cluster) postReconcileUpdateOSDProperties(desiredOSDs map[int]*OSDInfo) error {
+	osdUsage, err := cephclient.GetOSDUsage(c.context, c.clusterInfo)
```

Comment:
I think there can be a better way to query all osds

Reply:
This calls `ceph osd df`, it gets all the needed OSD properties including size, weight, and device class. What is your suggestion?

Reply:
Okay